packages/expo-cli/lib/test/lib/prepare-fixture.ts

```diff
@@ -0,0 +1,20 @@
+import { ncp } from 'ncp'
+import { mkdir } from 'fs'
+import rimraf from 'rimraf'
+import { promisify } from 'util'
+import { v4 as uuidv4 } from 'uuid'
+
+export const prepareFixture = async (fixture: string) => {
+  const tmp = `${__dirname}/../.tmp${uuidv4()}`
```

Comment:
could use built in crypto package here for a random (but not necessarily unique) value.